### PR TITLE
Merged 7437240d096dbec6086dea64efeb29b27e19458a

### PR DESCRIPTION
**Pull request by kipusoep**
I've merged #7437240d096dbec6086dea64efeb29b27e19458a as we discussed here: http://issues.umbraco.org/issue/U4-5491#comment=67-20105

### CODE REVIEW

**Comment by Shazwazza** [comment]
I think there was additional changes made to be consistent with the v7 stuff:

https://github.com/Umbraco/Umbraco-CMS/commit/311639332d9ec78bccc5d4e04156dfb8f655e92a

https://github.com/Umbraco/Umbraco-CMS/commit/0d74642e56d5f8333f0998296e5e57ea075a3498

**Comment by kipusoep** [comment]
How about this? It's the best I could do.

File: src/Umbraco.Web.UI/Umbraco.Web.UI.csproj [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <SccProvider>
     </SccProvider>
     <OldToolsVersion>4.0</OldToolsVersion>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI/Umbraco.Web.UI.csproj]: I hope it builds with v4.0 as Umbraco 6 still supports that.

Comment (kipusoep) [inline src/Umbraco.Web.UI/Umbraco.Web.UI.csproj]: Well it didn't build anymore until I changed this to v4.5

**Comment by kipusoep** [comment]
I'm running this code on a test environment and all seems to work well.

**Comment by hartvig** [comment]
Closing this as we don't maintain new builds of v6.

**Pull request closed without merging**